core/robstoll-lib/atrium-core-robstoll-lib-common/src/main/kotlin/ch/tutteli/atrium/core/robstoll/lib/reporting/DetailedObjectFormatter.kt

```diff
@@ -76,6 +76,7 @@ abstract class DetailedObjectFormatterCommon(
     private fun format(throwable: Throwable) = throwable::class.fullName
 
     private fun classNameAndIdentity(any: Any): String = INDENT + "(${any::class.fullName}${identityHash(" ", any)})"
+    private fun limitRepresentation(value: String, limit: Int = 10000) = if (value.length > limit) value.substring(0, limit) else value
```

Comment:
I would not provide `limit` as parameter as we don't use it (can still be introduce if necessary). Besides, default parameters create an extra class on the JVM byte-level which is hard to deal with regarding backward compatibility (because Kotlin does not give mechanisms to chose names for instance).

Reply:
Thanks for the explanation. This is good to know, i'll take it out